packages/bigtable/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google-cloud/bigtable",
-  "version": "0.8.0",
+  "version": "0.9.0",
```

Comment:
I don't mind this way as well ^, I kind of like it. But the script isn't there yet.